src/core/ts/background-scripts/background.js/telemetry/Telemetry.ts

```diff
@@ -304,7 +304,7 @@ export class Telemetry {
       notSupported.add(1);
       this.recordCommonMetadata(from, to);
     }, tabId);
-    this.updateInactivityTimerForAllTabs();
+    this.submitQueuedRecordings(tabId);
```

Comment:
is there an easy way to cover it in tests? if not, nvm

Reply:
Adding a new e2e test and verifying the collected telemetry is the way to go. I am speeding through issues currently without adding e2e tests everywhere due to resource constraints. This builds up technical debt so I created a general catch-all issue to address this: https://github.com/mozilla-extensions/bergamot-browser-extension/issues/201